### PR TITLE
Replaced links

### LINKED ISSUE #117: Replace links to pgijsbers.github.io to openml-labs.github.io
In the move from my personal account to this Github organization, all old links broke. For now I have forked the repo and built the docs there as well so old links work again, but since they still refer to my fork they should be updated to link back to this repo instead.

### PR DESCRIPTION
**Pull request by himanshu007-creator**
Resolves #117

### CODE REVIEW

**Comment by PGijsbers** [comment]
Hi @himanshu007-creator, thanks for helping out! There are also other links in the documentation that refer to the old location (i.e. https://github.com/PGijsbers/gama/) that need to be updated. Would you be willing to change those too?

**Comment by himanshu007-creator** [comment]
sure!

File: README.md [hunk 1]
```diff
@@ -21,20 +22,22 @@ In addition to its general use AutoML functionality, GAMA aims to serve AutoML r
...
 You can install GAMA with pip: `pip install gama`
 
 ## Minimal Example
+
 The following example uses AutoML to find a machine learning pipeline that classifies breast cancer as malign or benign.
-See the documentation for examples in 
+See the documentation for examples in
 [classification](https://pgijsbers.github.io/gama/master/user_guide/index.html#classification),
 [regression](https://pgijsbers.github.io/gama/master/user_guide/index.html#regression),
 using [ARFF as input](https://pgijsbers.github.io/gama/master/user_guide/index.html#using-arff-files).
```
Comment (PGijsbers) [inline README.md:39]: These are outdated as well.

File: docs/source/advanced_guide/configuration.inc [hunk 2]
```diff
@@ -1,59 +1,44 @@
...
-
-* ``class``, with a dictionary as value.
-
- The key specifies the algorithm, calling it should instantiate the algorithm.
- The dictionary specifies the hyperparameters by name and their possible values as list.
- All hyperparameters specified should be taken as arguments for the algorithm's initialization.
- A hyperparameter specified at the top level of the dictionary can share a name with a hyperparameter of the algorithm.
- To use the values provided by the shared hyperparameter, set the possible values to an empty list.
- If a list of values is provided instead, it will not use the shared hyperparameter values.
+:orphan: .. _search_space_configuration: GAMA Search Space Configuration
```
Comment (PGijsbers) [inline docs/source/advanced_guide/configuration.inc:1]: Based on the Github UI, it looks like you might have removed the newlines in this file? Please restore the line endings so the source is also readable.
image

**Review by PGijsbers — CHANGES_REQUESTED** [review]
Thanks, almost there! It looks like you inadvertently might have removed some newlines.

File: docs/source/advanced_guide/configuration.inc (hunk 2 above)
Comment (himanshu007-creator) [reply docs/source/advanced_guide/configuration.inc]: Done

File: README.md (hunk 1 above)
Comment (himanshu007-creator) [reply README.md]: Done

**Comment by PGijsbers** [comment]
Thanks! I think it looks good. I'll build the docs tomorrow to verify and merge if it's all good 👍

**Comment by PGijsbers** [comment]
Thank you very much 🎉

**Comment by himanshu007-creator** [comment]
Welcome @PGijsbers

**Pull request merged**